File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnStudioPickImportPluginApiId.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnPluginApiMethod;
import com.neome.java.api.meta.base.Types.MetaIdPlugin;
import com.neome.java.api.meta.base.Types.PluginApiId;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnStudioPickImportPluginApiId extends DefnFieldEditable
{
  @Nullable
  public PluginApiId[] excludePluginApiIdSet;

  @Nullable
  public EnumDefnPluginApiMethod[] filterApiMethodSet;

  public MetaIdPlugin metaIdPlugin;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnStudioPickImportPluginApiId.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnPluginApiMethod
import com.neome.api.meta.base.Types.MetaIdPlugin
import com.neome.api.meta.base.Types.PluginApiId

interface DefnStudioPickImportPluginApiId : DefnFieldEditable {
    val excludePluginApiIdSet: List<PluginApiId>?
    val filterApiMethodSet: List<EnumDefnPluginApiMethod>?
    val metaIdPlugin: MetaIdPlugin
}
